### DISCUSSION TITLE
How to automatically start the tour instead of clicking the beacon first?

### DISCUSSION THREAD

**Discussion by madhurgarg71** [gilbarbara/react-joyride#698] [Q&A] [answered]
In the official [codesandbox](https://codesandbox.io/s/github/gilbarbara/react-joyride-demo) example (controlled demo) the tour automatically starts as soon as the screen is loaded, how?
In my case it always starts with a beacon and then I have to click on that beacon to start the tour.

**Comment by madhurgarg71** [comment, marked as answer]
I just saw it's a duplicate question and I have got my answer 👍

**Reply by medmin** [reply]
Hi, would you please paste your answer here ? Thanks.

**Reply by dnorac** [reply]
https://github.com/gilbarbara/react-joyride/discussions/719#discussioncomment-1153032
